robot-core/src/test/resources/template.csv

```diff
@@ -1,5 +1,5 @@
-Label,Label Comment,ID,Synonyms,Type,Parent,Parts,Parts Annotation
-A rdfs:label,>AL rdfs:comment@en,ID,A IAO:0000118 SPLIT=|,CLASS_TYPE,C %,C part_of some %,>C rdfs:comment
```

Comment:
I would prefer for this unit test to stay the same as it was.

Reply:
Reverted